### DISCUSSION TITLE
Providing priorities for issues

### DISCUSSION THREAD

**Discussion by jakehader** [terrapower/armi#398] [Q&A] [answered]
As I was going through the open issues, I was thinking that it would be nice to have a priority filter by either labels or to think about using the `Projects` functionality of GitHub to triage issues on a Kanban board.

What are the team's thoughts on this? Would this level of granularity help communicate which types of developments are being prioritized over others in the near term versus long term or is this not necessary at this point? Thoughts?

**Comment by bmingst** [comment]
I have always found priorities useful on open item tracking.  It allows effort to be focused on the most important (or most bang-for-the-buck).  Labels would be fine.  I would avoid adding an additional system or layer (Kanban) unless it has other beneficial uses.

**Comment by john-science** [comment]
@jakehader The only GitHub solution for this is Labels/Categories.  Right now we have ~40 open tickets and ~5 don't have any labels at all.  So we could be more diligent about labeling our tickets. We could also easily add a Label/Category for Low/Medium/High Priorities. 

A bit of history might help.

GitHub Issues was designed in direct response to Jira.  High-level managers, in theory, like Jira because you can have tickets-within-tickets-within-tickets, and they can assign "priority" values and "topics" and "teams" and "estimated time for completion", and a ton of other things to each ticket. If you have ever tried to review a thousand tickets from dozens of developers over several years, you can see why you'd want these features. They are there to create a workflow for management.

But GitHub Issues provides only the simplest, flat interface for ticket management. It is meant to be 'task centric' and not 'management centric'.  

Anyway, we could add Labels or Categories for priority if @ntouran / @youngmit want. 

(I might be in favor of adding a "High Priority" label, but not a "low" or "medium" one.)

**Reply by jakehader** [reply]
My understanding is that `Projects` now offer the Kanban functionality as well for bug/ticket prioritization within a repository or organization. https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards#templates-for-project-boards

**Reply by john-science** [reply]
@jakehader  I hadn't seen that before! This _has_ to be a Microsoft addition.

**Comment by ntouran** [comment, marked as answer]
I'm not opposed to getting fancier in that regard, but it may make more sense after we get more engagement from other developers. For now, I think labels are probably most appropriate given our level of open PM complexity.

**Comment by jakehader** [comment]
Thanks for the discussion all. For now we will just use the labels/tags on issues.

**Reply by john-science** [reply]
Agreed.

Though it looks like maybe the "Projects" tab is what you were looking for?

**Reply by jakehader** [reply]
Yeah, exactly. If we want to do it in the future then Projects are what could help us.

**Reply by john-science** [reply]
Maybe we should pick an example "Project" to use this feature for, and try it out. Nick mentioned that as an idea.

It _might_ be a great way to drive community involvement and be more open about our priorities. 

But the devil is in the details, and it's all about how well that feature is implement; will it be useful.
